tests/kafkatest/services/zookeeper.py

```diff
@@ -44,7 +44,7 @@ class ZookeeperService(KafkaPathResolverMixin, Service):
     }
 
     def __init__(self, context, num_nodes, zk_sasl = False, zk_client_port = True, zk_client_secure_port = False,
-                 zk_tls_encrypt_only = False, version=DEV_BRANCH):
```

Comment:
We need to replace all dev by 3.8 for this file

Reply:
Hi @chia7712, I replace all dev with 3.8 in this file and add a comment. Could you take a look again when you have time? Thank you.